src/AppInstallerRepositoryCore/Public/AppInstallerRepositorySearch.h

```diff
@@ -121,6 +123,9 @@ namespace AppInstaller::Repository
         // Gets the manifest of this package version.
         virtual Manifest::Manifest GetManifest() const = 0;
 
+        // Gets the source where this package version is from.

```

Comment:
>// Gets the source where this package version is from. [](start = 8, length = 54)

This comment is actually what I was thinking of, getting the *source*:

virtual std::shared_ptr<ISource> GetSource() const = 0;